storage/mwm_id.hpp

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "platform/country_file.hpp"
+
+#include "base/assert.hpp"
+#include "base/checked_cast.hpp"
+
+#include <cstdint>
+#include <map>
+#include <vector>
+
+namespace storage
+{
+using NumMwmId = std::uint16_t;
+
+class NumMwmIds final
```

Comment:
But even if mwm is locked via MwmHandle, it still may be MARKED_TO_DEREGISTER, which is the case when old mwm is going to be removed soon, as there's another mwm with bigger version.